src/index.ts

```diff
@@ -44,19 +46,28 @@ const loadMovies = async (searchTerm: string): Promise<void> => {
   }
 }
 
-async function searchMovies(searchTerm: string): Promise<void> {
-  if (searchTerm.length > 0) {
-    await loadMovies(searchTerm)
-  }
-}
-async function keyPressHandler(e: KeyboardEvent): Promise<void> {
+async function searchMovies(e: KeyboardEvent): Promise<void> {
   if (e.key === 'Enter') {
     const searchTerm = movieSearchBox.value.trim()
+    movieSearchBox.value = ''
     movieDisplay.innerHTML = ''
-    await searchMovies(searchTerm)
+
+    if (!searchTerm.length) {
+      e.preventDefault()
+      const errMsg = []
+      errMsg.push('Please enter a Movie')
+
+      if (errMsg.length > 0) {
+        errorElement.innerText = errMsg.join(',')
+        console.error('NO MOVIE ENTERED')
+      }
```

Comment:
What is the intent of this logic? Are you trying to display multiple errors stacking up? If so, I don't think you'd get that behavior. The only logic that pushes a new error message onto the `errMsg` array immediately follows the array instantiation; so the array will never grow.

Reply:
I originally thought that I may want to reuse this functionality somewhere else, but it just makes more sense to just post a singular message if there is no movie.